### PR TITLE
Modulesync

### PR DESCRIPTION
**Pull request by juniorsysadmin**
Perform a modulesync along with other minor changes

### CODE REVIEW

**Comment by juniorsysadmin** [comment]
There's a few Rubocop violations including a silly RSpec one, not sure yet on the best way to handle them..

File: lib/facter/gluster.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@
 volume_ports = {}
 
 binary = Facter.value('gluster_custom_binary')
-if not binary or not File.executable? binary
+# rubocop:disable Style/AndOr
```
Comment (juniorsysadmin) [inline lib/facter/gluster.rb]: Probably a really simple fix, but rubocop-ignoring for now

File: lib/facter/gluster.rb [hunk 2]
```diff
@@ -24,33 +26,29 @@
...
-          end
-          if vol_status == "Started" then
-            # if `gluster volume status` fails for some reason, it spits to stderr,
-            # so we suppress that.
-            status = Facter::Util::Resolution.exec("#{binary} volume status #{vol} 2>/dev/null")
-            if status =~ /^Brick/ then
-              volume_ports[vol] = status.scan(/^Brick [^\t]+\t+(\d+)/).flatten.uniq.sort
-            end
-          end
+  peer_count = Regexp.last_match[1].to_i if output =~ /^Number of Peers: (\d+)$/
```
Comment (juniorsysadmin) [inline lib/facter/gluster.rb]: Perl-style backtick references is a rubocop violation

File: lib/facter/gluster.rb [hunk 3]
```diff
@@ -24,33 +26,29 @@
...
+  if peer_count > 0
+    peer_list = output.scan(/^Hostname: (.+)$/).flatten.join(',')
+    # note the stderr redirection here
+    # `gluster volume list` spits to stderr :(
+    output = Facter::Util::Resolution.exec("#{binary} volume list 2>&1")
+    if output != 'No volumes present in cluster'
+      output.split.each do |vol|
+        info = Facter::Util::Resolution.exec("#{binary} volume info #{vol}")
+        # rubocop:disable Metrics/BlockNesting
```
Comment (juniorsysadmin) [inline lib/facter/gluster.rb]: There are a bunch of rubocop nesting violations throughout, have decided to ignore them for now. (This file is a good candidate for refactoring)

File: metadata.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "covermymeds-gluster",
-  "version": "999.999.999",
-  "author": "Puppet Community",
+  "name": "puppet-gluster",
+  "version": "1.0.4",
```
Comment (juniorsysadmin) [inline metadata.json]: (The last release in the previous namespace was 1.0.3)

**Comment by juniorsysadmin** [comment]
This Pull Request is ready for review/merge

**Comment by jyaworski** [comment]
Woohoo! Thanks @juniorsysadmin. This LGTM.

After this is merged, please add puppet-gluster to modulesync_config's module list.

**Comment by jyaworski** [comment]
Can this be rebased?

**Comment by juniorsysadmin** [comment]
> Can this be rebased?
Done, with fixups to tests of #35 . Failures are due to something else (possibly PUP-5743)

**Comment by jyaworski** [comment]
Merging. Please add this to modulesync_config.

**Comment by jyaworski** [comment]
Ah. Did you keep the secret in .sync.yml?

**Comment by juniorsysadmin** [comment]
@jyaworski It didn't have a .travis.yml or a .sync.yml

**Pull request merged**